pubtools/_quay/container_image_pusher.py

```diff
@@ -110,15 +110,19 @@ def run_tag_images(cls, source_ref, dest_refs, all_arch, target_settings):
             target_settings.get("tag_images_wait_time_increase", 10),
         )
 
-    def copy_source_push_item(self, push_item):
+    def copy_source_or_v1_push_item(self, push_item, is_source=None):
```

Comment:
I would prefer if you separated this into its own function. Retrofitting an existing function implies that this is a part of normal workflow, which it is not. Separating it would make it easier to remove in the future.